src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/SafeHandles.cs

```diff
@@ -12,22 +13,19 @@ namespace Internal.Cryptography.Pal.Native
     /// </summary>
     internal abstract class SafePointerHandle<T> : SafeHandle where T : SafeHandle, new()
     {
-        public SafePointerHandle()
+        protected SafePointerHandle()
             : base(IntPtr.Zero, true)
         {
         }
 
-        public sealed override bool IsInvalid
+        internal static T CreateInvalid()
         {
-            get
-            {
-                return handle == IntPtr.Zero;
-            }
+            return new T();
         }
 
-        public static T InvalidHandle
+        public sealed override bool IsInvalid
         {
-            get { return new T(); }
+            get { return handle == IntPtr.Zero; }
```

Comment:
I was already editing the code around this, and the change makes it more consistent with others elsewhere in the file. I'd rather change it as I notice it then not change it at all, which would be the effect of my skipping it "until later", and it's not worth subsequently trying to split the changes apart into multiple commits. I'm sorry if such whitespace changes makes it slightly harder to review.